libpod/container_internal_linux.go

```diff
@@ -2377,7 +2377,17 @@ func (c *Container) getHosts() string {
 
 		if depCtr != nil {
 			host := ""
+			if depCtr.config.NetMode.IsSlirp4netns() {
+				// When using slirp4netns, the interface gets a static IP
+				if slirp4netnsIP, err := GetSlirp4netnsIP(depCtr.slirp4netnsSubnet); err != nil {
+					logrus.Warnf("Failed to determine slirp4netnsIP: %v", err.Error())
+				} else {
+					return fmt.Sprintf("%s\t%s %s\n", slirp4netnsIP.String(), c.Hostname(), c.config.Name)
+				}
+			}
 		outer:
+			// Paul this is returning nothing, even though I know depctr is up and
+			// running with an IP Address in rootfull mode?
```

Comment:
I think you want to change c.cniHosts() and not the code here

Reply:
Fine, how do I change it.  It looks like depctr has no network in rootfull mode.

Reply:
seems to work for me, what are you expecting?
This code is for host.containers.internal, if you want to add the container/host names you need to do this at a different place before this.

Reply:
Where would you suggest?